### PR TITLE
Migrate Babelforce Source Connector to Low Code

### LINKED ISSUE #29259: Babelforce
https://docs.airbyte.com/integrations/sources/babelforce

### PR DESCRIPTION
**Pull request by topefolorunso**
## What
Migrating Source Babelforce to Low-Code CDK resolves #29259 

## How
Created the connector using the connector buider UI, then imported the manifest.yaml to the low code cdk.

## Recommended reading order
1. `manifest.yaml`

## 🚨 User Impact 🚨
*Are there any breaking changes? What is the end result perceived by the user?*

*For connector PRs, use this section to explain which type of semantic versioning bump occurs as a result of the changes. Refer to our [Semantic Versioning for Connectors](https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors) guidelines for more information. **Breaking changes to connectors must be documented by an Airbyte engineer (PR author, or reviewer for community PRs) by using the [Breaking Change Release Playbook](https://docs.google.com/document/d/1VYQggHbL_PN0dDDu7rCyzBLGRtX-R3cpwXaY8QxEgzw/edit).***

*If there are breaking changes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.*

## Pre-merge Actions
*Expand the relevant checklist and delete the others.*

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

**Comment by marcosmarxm** [comment]
Hello 👋 , right now Airbyte's integration tests are failing for this connector and I'm going to work during the week to fix them. Due a high number of contributions we have a backlog and your contribution will be review next week. We ask to you share the integration test from your side whatever possible and if you don't have credentials to run tests to get in touch with @RealChrisSean as soon as possible. Also some common missing steps contributors does migrating to low code are (and you can double check to make sure you implemented this): 
* adding paginator
* update the documentation changelog + metadata file and dockerfile version
* if the connector has `start_date` parameter you must implement incremental syncs

File: airbyte-integrations/connectors/source-babelforce/source_babelforce/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  - type: DeclarativeStream
+    name: calls
+    primary_key:
+      - id
+    retriever:
+      type: SimpleRetriever
+      requester:
+        type: HttpRequester
+        url_base: https://{{ config['region'] }}.babelforce.com/api/v2/
+        path: calls/reporting
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-babelforce/source_babelforce/manifest.yaml:19]: I think the correct path should be `calls/reporting/simple`. The one using today is returning a csv format https://api.babelforce.com/#e0e4ec76-c14a-407c-bd77-17bc9ad29ab0

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Small detail, after we're ready to merge

File: airbyte-integrations/connectors/source-babelforce/source_babelforce/manifest.yaml (hunk 1 above)
Comment (topefolorunso) [reply airbyte-integrations/connectors/source-babelforce/source_babelforce/manifest.yaml]: Fixed now @marcosmarxm

**Comment by topefolorunso** [comment]
> I think the correct path should be `calls/reporting/simple`. The one using today is returning a csv format [api.babelforce.com/#e0e4ec76-c14a-407c-bd77-17bc9ad29ab0](https://api.babelforce.com/#e0e4ec76-c14a-407c-bd77-17bc9ad29ab0)

path fixed now @marcosmarxm

**Comment by octavia-approvington** [comment]
Enough! Let's do this <br/> goooooo!

**Review by octavia-approvington — APPROVED** [review]

**Pull request merged**